pages/about/AboutSiteHandler.inc.php

```diff
@@ -22,6 +22,7 @@ class AboutSiteHandler extends Handler {
 	function __construct() {
 		parent::__construct();
 		AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON);
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_USER);
```

Comment:
You can add multiple components to the same call to `requireComponents` -- e.g.:

    AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON, LOCALE_COMPONENT_PKP_USER);